tests/samplers_tests/test_nsgaii.py

```diff
@@ -129,7 +138,7 @@ def constraints_func(trial: FrozenTrial) -> Sequence[float]:
         nonlocal constraints_func_call_count
         constraints_func_call_count += 1
 
-        return (trial.number,)
+        return (constraint_value,)
```

Comment:
In the current code, the case where `constraint_value` is different between trials is not tested, which was originally tested.
How about returning something like `(constraint_value + trial.number,)`?

Reply:
Thanks for pointing that out! I fixed the code. PTAL.